plugin/dnstap/dnstapio/io.go

```diff
@@ -38,7 +42,7 @@ func (dio *DnstapIO) Dnstap(payload tap.Dnstap) {
 	select {
 	case dio.queue <- payload:
 	default:
-		fmt.Println("[WARN] Dnstap payload dropped.")
+		dio.log.Println("[WARN] Dnstap payload dropped.")
```

Comment:
just use log.Println, no need for all this.

Reply:
log is os.Stderr, is it ok to log on standard error?